### ISSUE TITLE
navbar

### ISSUE THREAD

**Issue by natehill** [esmero/archipelago_subtheme#6]
need to fix bootstrap navbar for mobile, i broke it somewhere

**Comment by DiegoPino** [comment]
@natehill go back to the original theme and look at the navigation css. I also think you moved the media selectors?

**Comment by natehill** [comment]
@DiegoPino yeah I'll figure it out, I wasn't being particularly orderly, was making a 'frankentheme', should be ok to solve just making an issue here so i don't forget
